administration/src/application/components/forms/ApplicationForm.tsx

```diff
@@ -1,10 +1,19 @@
 import { SetState, useUpdateStateCallback } from '../../useUpdateStateCallback'
 import { ApplicationType, BlueCardApplicationInput, BlueCardEntitlementType } from '../../../generated/graphql'
 import SwitchDisplay from '../SwitchDisplay'
-import { FormControl, FormControlLabel, FormLabel, Radio, RadioGroup } from '@mui/material'
+import { Button, Checkbox, FormControl, FormControlLabel, FormGroup, FormLabel, Radio, RadioGroup } from '@mui/material'
 import { Form } from '../../FormType'
 import StandardEntitlementForm, { StandardEntitlementFormState } from './StandardEntitlementForm'
 import PersonalDataForm, { PersonalDataFormState } from './PersonalDataForm'
+import BasicDialog from '../BasicDialog'
+import { dataPrivacyBaseHeadline, dataPrivacyBaseText } from '../../../constants/dataPrivacyBase'
+import styled from 'styled-components'
+
+const PrivacyPolicyButton = styled(Button)`
+  text-transform: capitalize !important;
```

Comment:
Is important! requried here?

Reply:
for styled components yes, but i could pass it inline with the `sx` of mui.